frontend/src/pages/post/PostDetail.tsx

```diff
@@ -38,11 +46,21 @@ function PostDetailPage() {
   return (
     <div>
       <Post post={post} me={user} />
-      {user && <CommentSection username={user.username} postId={id} />}
-      {post?.comments && post?.comments.length > 0
-        ? <CommentList comments={post.comments} ownerId={post.ownerId._id} moderators={post.moderatorsIds} />
-        : <p>There are nothing here O_Q</p>
-      }
+      {user && (
```

Comment:
comments uppdateras inte automatisk när man lägger till, behövs en hard-refresh. Kan man få in en useEffect som löser det?